src/GenerateOfflineMap.cpp

```diff
@@ -905,11 +920,6 @@ try {
 		_EXCEPTIONT("No output mesh (--out_mesh) specified");
 	}
 
-	// Overlap mesh
-	if (strOverlapMesh == "") {
```

Comment:
You should still be doing this check for FV projections. Perhaps need to check whether this is bilin/delaunay context first?